dali/python/backend_impl.cc

```diff
@@ -1420,7 +1468,7 @@ PYBIND11_MODULE(backend_impl, m) {
 
   // DALIImageType
   py::enum_<DALIImageType>(types_m, "DALIImageType", "Image type\n<SPHINX_IGNORE>")
-    .value("RGB", DALI_RGB)
+    .value("`", DALI_RGB)
```

Comment:
?

Reply:
Just kind of a typo error. Overall this PR is WIP, not really ready for review, as I am going to make some changes.

Reply:
Such things tend to slip to the final PR unnoticed. Just wanted to highlight this.